image_classification/MAE/main_multi_gpu_finetune.py

```diff
@@ -27,14 +27,13 @@
 import paddle.distributed as dist
 from datasets import get_dataloader
 from datasets import get_dataset
-from transformer import build_mae_finetune as build_model
+from transformer import build_mae_pretrain as build_model
```

Comment:
This file is used to finetune model. Why use `build_mae_pretrain` to build model instead of `build_mae_finetune`

Reply:
This is now fixed, should use finetune.